tests/asyncapi-2.0/Operation Trait Object/valid-empty-object.yaml

```diff
@@ -16,7 +16,6 @@ channels:
               format: email
       traits:
         - $ref: "#/components/operationTraits/userSignedUpTrait"
-        - $ref: "#/components/operationTraits/userSignedUpDescTrait"
```

Comment:
no such trait so example fails because of unresolvable ref

Reply:
Good catch. Thanks!